### ISSUE TITLE
Create "system" for creating new courses

### ISSUE THREAD

**Issue by kantord** [kantord/LibreLingo#1919]
**Is your feature request related to a problem? Please describe.**
Right now it's rather hard to create a new course.

There's a lot of documentation. I'll leave it to your judgement to decide whether this documentation is good enough that they contain all the necessary information. Nevertheless, this still means that it's necessary to find and read this documentation.

A lot of people might want to create a course. It is reasonable to expect (especially at this point) that people should have to read some documentation, but it would be nice if the course creation process could be at least started alone.

**Describe the solution you'd like**

I would like an easy way to create a course. It does not mean that the course creation should be completely automated, but it should be at least possible to start the process alone. Some features needed:

- There should be a "create course" button somewhere
- It should be possible to find all existing courses. If the language pair already exists, we should instead offer the opportunity to collaborate to the existing course instead of creating a new one
- There should be a way to "submit" the new course. This latter does not need to actually create or deploy a course. But it should initiate the processed in some way.

An easy way to do this could be to add some kind of contact form like logic.

Basically when the course creation is "submitted" all the data should be either sent to my email address (along with the contact information) or a new github issue should be created and the author of the course should be tagged in it.

**Describe alternatives you've considered**
I considered a system that actually creates a git repo. However this sounds harder to implement and it's unclear how much this would even be necessary. We probably don't need an extremely sophisticated way right now for course creation and we should rather focus on making the course contribution process easy. More people should need to contribute to existing courses rather than creating new ones
